bciers/libs/testConfig/src/global.tsx

```diff
@@ -40,6 +41,14 @@ declare module "vitest" {
 // Extend the global expect object with the custom matchers from jest-dom
 expect.extend(matchers);
 
+vi.mock("react", async () => {
+  const actualReact = await vi.importActual<typeof import("react")>("react");
+  return {
+    ...actualReact,
+    useContext: useContext,
+  };
+});
+
```

Comment:
I would never have known that this would be necessary in order to use the `useContext` mock. Great solve!